cpp/src/slice2cpp/Gen.cpp

```diff
@@ -273,6 +273,12 @@ namespace
         return r;
     }
 
+    /// Returns a doxygen formatted link to the provided Slice identifier.
+    string cppLinkFormatter(string identifier)
+    {
+        return "{@link " + fixKwd(identifier) + "}";
```

Comment:
Apparently we weren't escaping names in C++ links...
Linking to an identifier that is a C++ keyword is a _very_ niche case, but still good to fix.